recurseFibonacci.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+int Fibnacci(int);
```

Comment:
Your only mistake is the warning that this causes! you misspelled Fibonacci and forgot the variable name. Aside from that, nicely done!